### ISSUE TITLE
need to make build w/dependencies available on CDN

### ISSUE THREAD

**Issue by jchris** [fauna/faunadb-js#112]
Currently you have to use browserify which is fine for many people but a basic build would be easy to provide.

**Comment by eaceaser** [comment]
wait we have one, you can grab it via bower, and its hosted here: https://github.com/fauna/faunadb-js-release !

**Comment by jchris** [comment]
I'll link to it from this repo's description

**Comment by jchris** [comment]
From the initial reporter:

I did find fauna/faunadb-js-release, that was the first thing I tried. But the version of faunadb.js there depends on a nodejs-type module system which is not supported in modern browsers.

So I installed bower and installed it from there thinking maybe that would merge the code and lose the modules, but nope. The faunadb.js in the bower_components also depends on modules.

Ideally, I'd like to see an option where can load from a CDN directly to the browser. Here's a sample of some code I where I do that now:

<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css"><script src="//cdnjs.cloudflare.com/ajax/libs/ag-grid/8.0.1/ag-grid.min.js"></script>

<script src="//cdnjs.cloudflare.com/ajax/libs/sugar/2.0.4/sugar.js"></script>

I don't mind uploading a copy of faunadb.js to my server and calling it from there rather than a CDN. But with your scripts dependencies on modules, I'm stuck.
Thanks for taking this seriously BTW. I'm eager to give it another shot if we get over this hurdle. I currently use DynamoDB, but Fauna has enough advantage to make me willing to switch.

**Comment by erickpintor** [comment]
I've opened a [PR](https://github.com/jsdelivr/jsdelivr/pull/17026) to jsdelivr.com. We just need to wait for their approval.

**Comment by thos37** [comment]
The problem as described by the initial reporter is that this build relies on modules, and this has not been solved just by sticking it on a CDN.  This requires a different build that doesn't rely on modules and that creates a global var.

**Comment by erickpintor** [comment]
Hi @thos37. Thanks for the feedback.

Which version of the driver are you using?

There was an issue on the initial CDN release fixed by #123. That fix got out at version `1.1.0` and it is suppose to export a global variable called `faunadb`.

```
<script src="//cdn.jsdelivr.net/faunadb/1.1/faunadb.js"></script>
```

Is that sufficient for your project?

**Comment by thos37** [comment]
Yes, I tested 1.1 and it works for me.  Thanks!
